service/frontend/workflowHandler.go

```diff
@@ -1787,6 +1788,10 @@ func (wh *WorkflowHandler) StartWorkflowExecution(
 		return nil, wh.error(errInvalidTaskStartToCloseTimeoutSeconds, scope)
 	}
 
+	if startRequest.GetDelayStartSeconds() <= 0 {
```

Comment:
Does delay start seconds allow 0?

Reply:
yes, I replaced this with "<", will update the pr